stablehlo/tools/StablehloTranslateMain.cpp

```diff
@@ -97,11 +97,35 @@ class StablehloTranslateInterpreterFallback
                                  stablehlo::Process *process) final {
     llvm::StringRef funcName = op.getParentOfType<func::FuncOp>().getSymName();
     if (auto customCall = dyn_cast<stablehlo::CustomCallOp>(op)) {
-      if (customCall.getCallTargetName() == "check.eq") {
+      auto callTarget = customCall.getCallTargetName();
+      if (callTarget == "check.eq") {
         auto status = evalCustomCallCheckEq(customCall, scope);
         return stablehlo::wrapFallbackStatus(
             std::move(status), funcName, "stablehlo.custom_call(@check.eq)");
       }
+      if (customCall->getNumOperands() == 2) {
+        // TODO: Replace with custom_call --> check dialect pass
+        auto runtimeLhs = scope.findTensor(customCall->getOperand(0));
+        auto runtimeRhs = scope.findTensor(customCall->getOperand(1));
+        if (callTarget == "check.expect_close") {
+          auto status =
+              stablehlo::check::evalExpectCloseOp(runtimeLhs, runtimeRhs, 0, 3);
```

Comment:
Can we have the constant like `0`, `3`, and `0.0001` as default arguments of `evalCustomCallCheckEq`?

Reply:
I think yes, but also think that can be a followup